Apps/WebClient/src/ClientApp/app/models/timelineEntry.ts

```diff
@@ -10,9 +10,9 @@ export enum EntryType {
 export default abstract class TimelineEntry {
   public readonly id: string;
   public readonly type: EntryType;
-  public readonly date: Date;
+  public readonly date?: Date;
```

Comment:
I think the date is not optional

Reply:
In medicationStatement and medicationStatementHistory models the dispensedDate of the medication is optional... do we want to change this to mandatory to match timelineEntry then?

Reply:
I believe so, the date is a must for all sorting and rendering. It must also be on the backed